packages/react-components/react-tabs/src/stories/TabListWithPanels.stories.tsx

```diff
@@ -0,0 +1,148 @@
+import { makeStyles, shorthands } from '@griffel/react';
+import * as React from 'react';
+import { SelectTabData, SelectTabEvent, Tab, TabList } from '../index';
+import { AirplaneRegular, AirplaneTakeOffRegular, TimeAndWeatherRegular } from '@fluentui/react-icons';
+import { TabValue } from '../Tab';
+import { tokens } from '@fluentui/react-theme';
+
+const useStyles = makeStyles({
+  root: {
```

Comment:
change to wrapperDiv or something that doesn't sound like a slot

Reply:
Seems like we `base` in lots of components.

Reply:
I will go back through all the tab stories to update how I name useStyles. (same as above issue)